lib/open_mic.rb

```diff
@@ -0,0 +1,33 @@
+class OpenMic
+
+  attr_reader :location, :date, :performers
+
+  def initialize(event_info)
+    @event_info = event_info
+    @location = event_info[:location]
+    @date = event_info[:date]
+    @performers = []
+  end
+
+  def welcome(performer)
+    @performers << performer
+  end
+
+  def all_jokes
```

Comment:
I love this as a helper method!  It is a great separation of responsibility.  One way you _could_ clean it up:

```
@performers.map do |performer|
  performer.jokes
end.flatten
```